GitUI/CommandsDialogs/Menus/CopyPathsToolStripMenuItem.cs

```diff
@@ -0,0 +1,69 @@
+﻿using GitCommands;
+using GitExtUtils;
+
+namespace GitUI.CommandsDialogs.Menus
+{
+    internal partial class CopyPathsToolStripMenuItem : ToolStripMenuItemEx
+    {
+        private Func<IEnumerable<string?>> _getSelectedFilePaths;
+
+        public CopyPathsToolStripMenuItem()
+        {
+            InitializeComponent();
+
+            copyFullPathsNativeToolStripMenuItem.Font = new(copyFullPathsNativeToolStripMenuItem.Font, FontStyle.Bold);
+        }
+
+        public CopyPathsToolStripMenuItem Initialize(Func<GitUICommands> getUICommands, Func<IEnumerable<string?>> getSelectedFilePaths)
+        {
+            Initialize(getUICommands);
+            _getSelectedFilePaths = getSelectedFilePaths;
+            return this;
+        }
+
+        private void CopyFullPathsNativeToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            ((ToolStripItem)sender).GetCurrentParent().Hide();
```

Comment:
Is there a reason we can't use `this.GetCurrentParent().Hide();`?

Reply:
No, there is none :)